web/index_template.html

```diff
@@ -51,6 +51,7 @@
       href="https://fonts.googleapis.com/css?family=Zilla+Slab:400,600"
       rel="stylesheet"
     />
+    <link rel="apple-touch-icon" href="./apple-touch-icon-150x150.png" />
```

Comment:
Can you use the same convention as the other static icon images in this file, i.e. using the static img file? Also, I don't think the filename itself has to be called apple-touch-icon or whatever, as long as it's specified in the `rel`. 

I've gotten a better icon for this from the design folks as well. 
![CV iOS App Icon](https://user-images.githubusercontent.com/582683/88215041-1538f080-cc29-11ea-8e71-3753f7a40120.png)

Reply:
Hmmm...I dont know what you mean by 'static img file'. Is this what you mean?

`<link rel="apple-touch-icon" href="https://voice.mozilla.org/img/voice-touch-icon.png" />`

Reply:
Yep exactly. We're using the `/img` folder for static meta image assets, so let's stick to the convention and avoid unnecessary files on the root level.